erfa/ufunc.c.templ

```diff
@@ -376,7 +376,7 @@ ufunc_loop_matches(PyUFuncObject *self,
          */
         if (types[i] == NPY_VOID && dtypes != NULL) {
             int op_descr_type_num = op_descr->type_num;
-            int dtype_elsize = dtypes[i]->elsize;
+            int dtype_elsize = PyDataType_ELSIZE(dtypes[i]);
```

Comment:
```suggestion
            npy_intp dtype_elsize = PyDataType_ELSIZE(dtypes[i]);
```
Probably also still wrong in NumPy :).  Not that it matters in practice, I don't think NumPy _actually_ allows that large voids yet.  Just setting the stage.

Reply:
of course ! I lost this bit while cleaning the branch, thanks for catching my mistake !